cmd/applyconfig/logrusutil/logrusutil.go

```diff
@@ -0,0 +1,47 @@
+package logrusutil
```

Comment:
Remove this when https://github.com/kubernetes/test-infra/pull/14889 gets merged.